src/components/EntityInfo/EntityMenu.vue

```diff
@@ -186,25 +186,33 @@
             </v-tooltip>
 
             <!-- Request AGM Location Change -->
-            <v-tooltip
-              right
-              content-class="right-tooltip"
-            >
-              <template #activator="{ on }">
-                <v-list-item
-                  v-if="enableAgmLocationChg"
-                  id="agm-loc-chg-list-item"
-                  :disabled="!isAllowed(AllowableActions.AGM_LOCATION_CHG)"
-                  v-on="on"
-                  @click="goToAgmLocationChgFiling()"
-                >
-                  <v-list-item-title>
-                    <span class="app-blue">Request AGM Location Change</span>
-                  </v-list-item-title>
-                </v-list-item>
-              </template>
-              Request an AGM location change.
-            </v-tooltip>
+            <div>
+              <v-tooltip
+                right
+                content-class="right-tooltip"
+              >
+                <template #activator="{ on }">
+                  <div
```

Comment:
wrap in a `div` to show tooltip when list item is disabled

Reply:
Looks good:

In good standing:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/b3483d87-f799-4271-9fa4-9a2bb58fa648)

Not in good standing:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/aa8b16b7-1e0c-4e2f-9880-7f392026be12)